consensus-types/blocks/execution.go

```diff
@@ -723,6 +723,17 @@ func IsEmptyExecutionData(data interfaces.ExecutionData) (bool, error) {
 		}
 	}
 
+	ws, err := data.Withdrawals()
```

Comment:
im not sure i understand if this breaks bellatrix. What if we have a bellatrix block that has execution data? This function will return false, error in this case. Shouldn't this only apply for capella? it feels a bit weird to use withdrawals as a proxy to verify if execution data is empty

Reply:
This doesn't break Bellatrix. There are passing tests for the fork. The reason is that in Bellatrix fetching withdrawals will result in `ErrUnsupportedGetter`, which does nothing and jumps out of the `switch` statement (in Go switch statements do not fall through). Maybe we should be more explicit and use `break` instead of having an empty case?